### PR TITLE
Remove test delays

### PR DESCRIPTION
**Pull request by davimacedo**
According to the discussion on https://github.com/parse-community/parse-server/pull/5573 this PR removes some test delays.

### CODE REVIEW

File: spec/ParseLiveQueryServer.spec.js [hunk 1]
```diff
@@ -158,12 +158,10 @@ describe('ParseLiveQueryServer', function() {
...
-        __indexBuildCompletionCallbackForTests: promise => {
-          promise.then(() => {
-            expect(parseServer.liveQueryServer).not.toBeUndefined();
-            expect(parseServer.liveQueryServer.server).toBe(parseServer.server);
-            parseServer.server.close(() => done());
-          });
+        serverStartComplete: () => {
+          expect(parseServer.liveQueryServer).not.toBeUndefined();
+          expect(parseServer.liveQueryServer.server).toBe(parseServer.server);
+          parseServer.server.close(() => done());
```
[the remark this answers is not in the document]
Comment (davimacedo) [reply spec/ParseLiveQueryServer.spec.js]: just changed

File: spec/ParseLiveQueryServer.spec.js [hunk 2]
```diff
@@ -181,15 +179,13 @@ describe('ParseLiveQueryServer', function() {
...
-            );
-            parseServer.liveQueryServer.server.close(() => {
-              parseServer.server.close(() => done());
-            });
+        serverStartComplete: () => {
+          expect(parseServer.liveQueryServer).not.toBeUndefined();
+          expect(parseServer.liveQueryServer.server).not.toBe(
+            parseServer.server
+          );
+          parseServer.liveQueryServer.server.close(() => {
```
[the remark this answers is not in the document]
Comment (davimacedo) [reply spec/ParseLiveQueryServer.spec.js]: just changed to parseServer.liveQueryServer.server.close(parseServer.server.close.bind(parseServer.server, done))

**Comment by acinader** [comment]
Ready to merge?

**Comment by davimacedo** [comment]
Yes. I think it is.

**Pull request merged**